libcontainer/seccomp/seccomp_linux.go

```diff
@@ -0,0 +1,165 @@
+// +build linux,cgo,seccomp
+
+package seccomp
+
+import (
+	"fmt"
+	"log"
+	"syscall"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
+	libseccomp "github.com/seccomp/libseccomp-golang"
+)
+
+var (
+	actAllow = libseccomp.ActAllow
+	actTrap  = libseccomp.ActTrap
+	actKill  = libseccomp.ActKill
+	actErrno = libseccomp.ActErrno.SetReturnCode(int16(syscall.EPERM))
```

Comment:
Can be made configurable pretty easily, if there's a demand for it. Though it would require modification to the API to let users configure the return code.